lib/auth/googleauth.js

```diff
@@ -37,9 +37,9 @@ function GoogleAuth() {
 }
 
 // Executes the given callback if it is not null.
-function callback(c, err, res) {
+function callback(c, err, arg1, arg2) {
   if (c) {
-    c(err, res);
+    c(err, arg1, arg2);
```

Comment:
Are we sure we want to do this? Won't it wipeout the user's context if they bound their callback to something? e.g.

```js
function callback(c) {
  if (c) {
    c.apply(null, Array.prototype.slice.call(arguments, 1));
  }
}
```